conan/tools/premake/premake.py

```diff
@@ -0,0 +1,19 @@
+class Premake(object):
+    def __init__(self, conanfile):
+        self._conanfile = conanfile
+
+    # automatically chooses premake action based on used compiler
+    def configure(self):
+        if "Visual Studio" in self.settings.compiler:
```

Comment:
It is likely that ``msvc`` will be the compiler that survive in 2.0. Please try to add support for it too.